epochStart/mock/storerMock.go

```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
 	"fmt"
 	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/storage"
```

Comment:
We shouldn't import the `storage` package which is quite heavy only for the simple struct `KeyValuePair` which only contains `Key` and `Value`. Maybe move `KeyValuePair` to `common` package with a TODO to move it to core

Reply:
Fixed, moved to `common`. Added TODO.